cy_canvas/containers/CyReact.js

```diff
@@ -82,10 +82,14 @@ const Cy = React.createClass({
     var node = evt.cyTarget;
     this._cxtdragStart = true;
     this._edgeFrom = node;
+    this._edgeType = node.hasClass("user_says")? "us2r": "r2us"
```

Comment:
I know what us2r and r2us are and I also get the idea that it's easier to type. However, it's not intuitive and I'm not sure people who read this line of code can instantly figure out the connection between the abbreviations and the actual meanings.